### ISSUE TITLE
Weights of models trained with TSDAE

### ISSUE THREAD

**Issue by xboldt** [UKPLab/sentence-transformers#3067]
Hi SBert Team,

Have you released any models (encoder and decoder) that were pretrained or extensively fine-tuned using the TSDAE loss? I’m particularly interested in the decoder weights from the `DenoisingAutoEncoderLoss` class after training.

Thanks!

**Comment by tomaarsen** [comment]
Hello!

I don't believe we have any models under https://huggingface.co/sentence-transformers that have been trained with TSDAE. There's a few public models: https://huggingface.co/models?library=sentence-transformers&search=tsdae, but it seems that they only include the encoder - the decoder is not usually saved it seems. 

- Tom Aarsen

**Comment by xboldt** [comment]
Thanks for getting back to me @tomaarsen. Unfortunately, I have only ever been able to find the trained encoders as well. 

@kwang2049 any chance you could make the decoders from the TSDAE paper https://arxiv.org/pdf/2104.06979 available?
